apps/web/src/features/myAccounts/components/FilteredSafes/index.tsx

```diff
@@ -15,6 +18,12 @@ const FilteredSafes = ({
 }) => {
   const filteredSafes = useSafesSearch(allSafes ?? [], searchQuery)
 
+  useEffect(() => {
+    if (searchQuery) {
+      trackEvent({ category: OVERVIEW_EVENTS.SEARCH.category, action: OVERVIEW_EVENTS.SEARCH.action })
```

Comment:
Not critical but this event is tracked as a "customClick" event instead of a meta event.